code/src/main/java/com/onlinecontacttracing/servlets/get-data/GetNegativeUserInfoServlet.java

```diff
@@ -0,0 +1,21 @@
+package com.onlinecontacttracing.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/get-negative-user-info")
+public class GetNegativeUserInfoServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    try {
+      request.getRequestDispatcher("/get-negative-user-calendar-info").forward(request,response);
+    } catch(Exception e) {
+      e.printStackTrace();
+    }
+    System.out.println("Negative User done getting info");
```

Comment:
How about "Done getting negative user info"

Reply:
Also I don't think it should print that if there is an exception

Reply:
This print statements are just for debugging. Once we implement the actual servlets we can remove them

Reply:
Let's either log, or put it in the response. System.out.println will be hard to find in the stack traces, where as a log will be obvious and a response makes it easy to validate.